### PR TITLE
Add blazor to getbrowsertimingheader api

### PR DESCRIPTION
**Pull request by nr-ahemsath**
The .NET agent had a customer wanting to use the GetBrowserTimingHeader() API in a Blazor Webassembly application.  Unfortunately we don' t support that scenario.  This PR updates our API doc to call out that incompatibility, as well as provide correct instructions for how to use the API for a Blazor Server application, which we do support.

I also changed the code type for the Razor (not Blazor) instructions to be cshtml as that is the actual file type in that case.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent-api.mdx [hunk 1]
```diff
@@ -113,7 +113,39 @@ An HTML string to be embedded in a page header.
...
+```
+
+### With Blazor [#blazor]
+
+<Callout variant="important">
+  This API is not supported for Blazor Webassembly, because the agent is unable to instrument Webassembly code.  The following examples are for Blazor Server applications only.  Use the [copy-paste method](https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#copy-paste-app) of adding the Browser Agent to Blazor Webassembly pages.
+</Callout>
+
+<Callout variant="important">
+  This API can not be placed in a `<HeadContent>` element of a .razor page.  Instead, it should be called from _Layout.cshtml or an equivalent layout file.
```
Comment (akristen) [inline src/content/docs/apm/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent-api.mdx:134]:
```suggestion
  This API can not be placed in a `<HeadContent>` element of a `.razor` page.  Instead, it should be called from `_Layout.cshtml` or an equivalent layout file.
```

Comment (akristen) [reply src/content/docs/apm/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent-api.mdx:134]: @nr-ahemsath Not sure if these should be braced for code block, but added it here

Comment (nr-ahemsath) [reply src/content/docs/apm/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent-api.mdx]: That works, thanks for doing that.

**Review by akristen — APPROVED** [review]
Thanks!

**Pull request merged**